packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -65,6 +66,18 @@ export default class UpdateObject extends BaseAction {
 				message: this.translationService.getMessage(ERRORS.NO_ACTIVE_WORKSPACE),
 			};
 		} else {
+			const projectFolderPath = this.getProjectFolderPath();
+			const projectInfoService = new ProjectInfoServive(projectFolderPath);
+			if (projectInfoService.isAccountCustomizationProject() || projectInfoService.isSuiteAppProject()) {
```

Comment:
I think we may be missing a try/catch block to avoid problems when for instance we have malformed manifest.xml

Reply:
Added.